app/assets/javascripts/Components/Result/submission_file_panel.jsx

```diff
@@ -72,6 +72,10 @@ export class SubmissionFilePanel extends React.Component {
       }
       this.setState({selectedFile});
     }
+
+    if (prevProps !== this.props) {
```

Comment:
Hm I don't think it's good to do a full prop comparison. Perhaps just the `result_id`?

Reply:
Comparing `props.annotations` because that is the actual prop change that should trigger a re-mount